src/services/WebViews.ts

```diff
@@ -185,6 +201,42 @@ export class WebViews {
 			this.log.error(err);
 		}
 	}
+
+	private async assignUser(assignUser: any, rows: any) {
```

Comment:
Add code comments that describes the way functionality is implemented.

Reply:
Done